### ISSUE TITLE
Support fusing LayerNormalization without mean centering (aka RMSNorm)

### ISSUE THREAD

**Issue by robertknight** [robertknight/rten#495]
The [T5](https://huggingface.co/google-t5/t5-small) and Llama models use [RMSNorm](https://pytorch.org/docs/stable/generated/torch.nn.modules.normalization.RMSNorm.html) aka. "simplified layer normalization" which omits the mean centering step. The graph optimizer should recognize and fuse this. The existing implementation of LayerNormalization can accommodate this with some minor tweaks.

The subgraph seen in the T5 and Llama 3.2 models looks like so:

<img width="500" alt="rmsnorm-subgraph" src="https://github.com/user-attachments/assets/c75de562-bad6-45cd-88ec-ae4205a332ae" />

Other than omitting the mean centering step, another difference is that the division in `x / sqrt(rms(x) + epsilon)` is represented as multiplication by reciprocal here.
